lib/widgets/app_bar.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart' as M;
+import 'package:flutter/cupertino.dart' as C;
+
+class AppBar {
+  final Widget title;
+
+  AppBar({this.title});
+
+  C.CupertinoNavigationBar toCupertino() {
```

Comment:
Not a big deal, but I think `asCupertino` and `asMaterial` would render a better name, what do you think?

Reply:
Good idea!